my_dual_robot_cell/my_dual_robot_cell_control/config/combined_controllers.yaml

```
@@ -0,0 +1,244 @@
+controller_manager:
+  ros__parameters:
+    alice_joint_state_broadcaster:
+      type: joint_state_broadcaster/JointStateBroadcaster
+
+    bob_joint_state_broadcaster:
+      type: joint_state_broadcaster/JointStateBroadcaster
```

Only one joint_state_broadcaster should be sufficient.
```suggestion
    joint_state_broadcaster:
      type: joint_state_broadcaster/JointStateBroadcaster
```